pyspark/bigdl/util/engine.py

```diff
@@ -41,6 +41,16 @@ def check_spark_source_conflict(spark_home, pyspark_path):
         warnings.warn(warning_msg)
 
 
+def __sys_path_insert(file_path):
+    try:
+        if file_path not in sys.path.split(":"):
+            print("Prepending %s to sys.path" % file_path)
+            sys.path.insert(0, file_path)
+    except:
```

Comment:
Why will this raise an exception? In the try block even if file_path is in sys.path, I suppose it won't throw an error but just not go into the if branch?

Reply:
I mean just add the if condition as well when inserting to sys.path...